.changeset/ember-update-example.md

```diff
@@ -0,0 +1,5 @@
+---
+"examples": patch
```

Comment:
Hmm, I guess so since examples aren't published to npm. I'll fix it <picture data-single-emoji=":blob_thumbs_up:" title=":blob_thumbs_up:"><img class="emoji" width="20" height="auto" src="https://emoji.slack-edge.com/T0CAQ00TU/blob_thumbs_up/f3fed0d24f5f9a1f.png" alt=":blob_thumbs_up:" align="absmiddle"></picture>

Reply:
Fixed in [95ca1e2](https://github.com/vercel/vercel/pull/10347/commits/95ca1e24dd17cb9a7550c9b238677f4dee0fbcb0)